packages/block-editor/src/components/link-control/search-create-button.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button, Icon } from '@wordpress/components';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+
+export const LinkControlSearchCreate = ( {
+	searchTerm = '',
```

Comment:
I guess we decided the default here would handle the case where otherwise it would output `New page: <mark>undefined</mark>` ? Because I wonder instead if we should be rendering anything at all. In other words, allow it be left `undefined`, then the component can return `null` rather than a button which is unusable at best (posts can't be created without at least a title, content, or excerpt, so it would error even if we allowed it to continue to exist).

Reply:
Ah great spot. This (return `null`) was exactly what the component originally did. However, there was a brief interlude where we wanted the "Create" button to be able to create blank pages. I guess when I removed that functionality I forgot to reinstate the check. My bad.

Will fix.

Reply:
Fixed.